README.md

```diff
@@ -19,69 +19,101 @@ pip install .
 ## Implemented operations
```

Comment:
Should this read "Implemented and planned operations"? And mark the different operations as "implemented" or "planned".

Reply:
Good point, let's make them all planned for now